spritesheetExporter/uispritesheetexporter.py

```diff
@@ -145,108 +154,131 @@ def addDescribedWidget(self, parent, listWidgets, align = Qt.AlignLeft):
         parent.addLayout(layout)
         return layout
 
-
     def initialize_export(self):
-
         # putting stuff in boxes
         # and boxes in bigger boxes
         self.exportName.setText(self.exp.exportName)
-        self.addDescribedWidget(parent = self.topLayout, listWidgets = [
-        describedWidget(
-        widget = self.exportName,
-        descri = "Export name:",
-        tooltip = "The name of the exported spritesheet file")])
-        self.addDescribedWidget(parent = self.topLayout, listWidgets = [
-        describedWidget(
-        widget = self.exportDirTx,
-        descri = "Export Directory:",
-        tooltip = "The directory the spritesheet will be exported to")])
+        self.addDescribedWidget(parent=self.topLayout, listWidgets=[
+            describedWidget(
+                widget=self.exportName,
+                descri="Export name:",
+                tooltip="The name of the exported spritesheet file")])
+        self.addDescribedWidget(parent=self.topLayout, listWidgets=[
+            describedWidget(
+                widget=self.exportDirTx,
+                descri="Export Directory:",
+                tooltip="The directory the spritesheet will be exported to")])
 
         self.exportDir.addWidget(self.exportDirButt)
         self.exportDir.addWidget(self.exportDirResetButt)
         self.topLayout.addLayout(self.exportDir)
 
-
-        #self.outerLayout.addItem(self.spacer)
-        self.addDescribedWidget(parent = self.topLayout, listWidgets = [
-        describedWidget(
-        widget = self.customSettings,
-        descri = "Use Custom export Settings:",
-        tooltip = "Whether to set yourself the number of rows, columns, \nfirst and last frame, etc. (checked) \nor use the default values (unchecked) ")])
+        # self.outerLayout.addItem(self.spacer)
+        self.addDescribedWidget(parent=self.topLayout, listWidgets=[
+            describedWidget(
+                widget=self.customSettings,
+                descri="Use Custom export Settings:",
+                tooltip="" +
+                "Whether to set yourself the number of rows, columns, \n" +
```

Comment:
Note: on this line and the others like it, I believe that the space before `\n` is useless, as the user cannot see it. I would recommend removing it -- but I didn't to keep the code exactly equivalent to the one before.

Reply:
it is indeed useless. Thanks! I'll remove it.